### ISSUE TITLE
Update/Add/Rename EX Folder Subtypes (NN)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#4469]
### Currently Management is requesting (once or twice a week) for reports on how many requests are received per permit and/or customer type. In order to make it a more efficient and information report, we'd like to update EX folder subtypes so that permit applicants are more likely to select the correct permit subtype.

**Stakeholders:** Paloma, Kim
**DTS Manager:** Nate

**Acceptance Criteria:**
- Changes Apply to the EX folder only and are stated in the table below:

SubType   Name | SubCode | Request | Original | New
-- | -- | -- | -- | --
Austin Energy Electric | 50693 | Name Change | Austin Energy Electric | Austin Energy (AE)
Austin Transportation | 50694 | Name Change | Austin Transportation | Austin Transportation (ATD)
City Utility AWU/STORM | 50688 | Name Change | City Utility AWU/STORM | Austin Water Utility (AWU)
Austin Energy Chilled Water (OSER) | 50692 | Retire | N/A | N/A
Street/Bridge | 50695 | Retire | N/A | N/A
Corridor Program Office | NEEDED | Add | N/A | Corridor Program Office
Project Connect | NEEDED | Add | N/A | Project Connect
Private Contractors | 50540 | Name Change | Private Contractor | Private Development
Public Works/Electric | 50690 | Name Change | Public Works/Electric | Public Works Department PWD
Watershed Protection Storm Drain | 50696 | Name Change | Watershed Protection Storm Drain | Watershed Protection Department WPD
  |   |   |   |  
  |   |   | New WorkType |  
Franchised Utility | 50685 | Add 3 Worktypes | Gas Utility |  
  |   |   | Other Electric Utility |  
  |   |   | Telecommunication/Cable Utility |

**Comment by Nadin-Nader** [comment]
@TracyLinder Scheduled the scoping meetings for this. First one will/may not have Kim since she may be on leave.

Scoping meeting Scheduled for:
3/2/2021 @10am-1030am
3/23/2021 @10am-1030am Follow Up Meeting

**Comment by Nadin-Nader** [comment]
Had the meeting with Paloma and Tracy and she explained that this change is needed as she gets asked for reports on how many permits are requested per customer type. We were able to hash out the Acceptance Criteria. We are just waiting on go ahead from CTM to make changes in AMANDA and for Paloma to let us know if the Corridor Program Office (CPO) and the Small Cell Node have been agreed upon.

1. Applies to EX folder only

- [ ] Subtypes that need a name change:
     Austin Energy Electric 50693 Change to Austin Energy (AE)
     Austin Transportation 50694 Add ATD
     City Utility AWU/STORM 50688 Rename to Austin Water Utility (AWU)

- [ ] Subtypes to be Retired:
     Austin Energy Chilled Water (OSER)50692 Retire
     Street/Bridge 50965 Retire

- [ ] Split Subtype Franchised Utility 50685 to have 3 options:
     Gas Utility
     Other Energy/Electric Utility
     Telecommunication/Cable Utility

- [ ] Subtype to be Added (Still need confirmation):
     Corridor Program Office (CPO)
     Small Cell Node

**Comment by Nadin-Nader** [comment]
@TracyLinder does this look ok?

**Comment by Nadin-Nader** [comment]
Moving to Backlog

**Comment by TracyLinder** [comment]
@Nadin-Nader please finish the spreadsheet & add it to the description.
[Subcode.xlsx](https://github.com/cityofaustin/atd-data-tech/files/6158140/Subcode.xlsx)

**Comment by Nadin-Nader** [comment]
I updated the spreadsheet. We are just waiting on Paloma and CPO to approve the last 2 items.

Subha will be looking into this one.

**Comment by SubhaShroff** [comment]
The necessary changes are done except the last two items. A commit is done for the changes and DEV promotion request is raised: **SCTASK0130608**

**Comment by Nadin-Nader** [comment]
## **Status:**
Waiting on CTM DevOps to deploy to DEV for now (**SCTASK0130608**)

**Comment by SubhaShroff** [comment]
**Status:** Dev Deployment is done today(4/13/2021). I have to test it in DEV.

**Comment by TracyLinder** [comment]
@Nadin-Nader the spreadsheet/chart above doesn't mention subtypes PVT, Public Works/Electric, Watershed Protection Storm Drain
- Public Works/Electric: Update name to 'Public Works Department PWD
- Watershed Protection Storm Drain: Update name to 'Watershed Protection Department WPD'

Can you please update the issue. Thank you.

**Comment by SubhaShroff** [comment]
**Status:**  

Corridor Program Office | NEEDED | Add | N/A | Corridor Program Office
-- | -- | -- | -- | --
Small Cell Node | NEEDED | Add | N/A | Small Cell Node

Should I add the above?

**Comment by Nadin-Nader** [comment]
@TracyLinder @SubhaShroff I have updated the list

**Comment by SubhaShroff** [comment]
@Nadin-Nader @TracyLinder  DEV deployment request is submitted for the updated AC:  SCTASK0133028 for commit-3c6157039514.

**Comment by TracyLinder** [comment]
Per Paloma:
- we should not add Small Cell Node
- Add a subtype for Project Connect

@Nadin-Nader please update the table above. Thanks!

**Comment by Nadin-Nader** [comment]
@SubhaShroff  the list has been updated to show that Small Cell is not needed and Project connect needs to be added.

**Comment by Nadin-Nader** [comment]
## **Status:**
Waiting on DevOps to Deploy the 2nd part to DEV **(SCTASK0133028 for commit-3c6157039514)

**Comment by SubhaShroff** [comment]
**Status:** 

1. (SCTASK0133028 for commit-3c6157039514) is DEV deployed today. 
2. Another commit- 587e48a2d28e(to remove Small Cell Node and add Project Connect) was done yesterday. Once @Nadin-Nader approves it, I will submit DEV deployment request.

**Comment by SubhaShroff** [comment]
**Status:**  Dev Deployment for the commit-_587e48a2d28e_ is submitted: _SCTASK0134113_

**Comment by SubhaShroff** [comment]
![image](https://user-images.githubusercontent.com/51680715/116741116-85944f80-a9bb-11eb-839f-05577d9aaa4a.png)
![image](https://user-images.githubusercontent.com/51680715/116741147-8fb64e00-a9bb-11eb-8391-b97c6672c64f.png)

@TracyLinder  @Nadin-Nader Tested it in DEV, subtypes are updated as in AC. Please refer the above screenshots.

**Comment by TracyLinder** [comment]
### Update
- Stakeholders approve based on the screenshots above. Please promote up.

**Comment by SubhaShroff** [comment]
@TracyLinder @Nadin-Nader  TEST Deployment request is submitted : **SCTASK0134702**

**Comment by SubhaShroff** [comment]
SCTASK0134702 shows closed status. I tested it on TEST, but no changes are reflecting. I am not sure if the changes will show on the next day.

**Comment by SubhaShroff** [comment]
@Nadin-Nader @TracyLinder Tested in TEST. Looks as expected.
![image](https://user-images.githubusercontent.com/51680715/117013182-3ef16e80-acb5-11eb-800c-48e651edb350.png)
![image](https://user-images.githubusercontent.com/51680715/117013218-4749a980-acb5-11eb-9ccb-e9511790e13b.png)

**Comment by Nadin-Nader** [comment]
@TracyLinder This is on ## **TEST** and is ready for Stakeholder UAT. I will schedule UAT with Stakeholders

**Comment by Nadin-Nader** [comment]
UAT on TEST Scheduled for May 11, 2021 at 930am to move this change to PROD

## **Status:**
Waiting on meeting to UAT on TEST

**Comment by Nadin-Nader** [comment]
This was UATd in todays ROW Meeting and it passed. It is ready to go to PROD once #6021 is done. #6021 is for the same changes but for them to be sown on Portal.

## **Status:**

Waiting on #6021 to be completed before moving to PROD

**Comment by SubhaShroff** [comment]
### Status:
Waiting on #6021 to be completed. #6021 is waiting for DEV deployment.

**Comment by SubhaShroff** [comment]
### Status:
Waiting on #6021 to be completed. #6021 is done on DEV, will move to TEST with feedback.

**Comment by SubhaShroff** [comment]
@Nadin-Nader **SCTASK0140831** is submitted for PROD deployment.

**Comment by SubhaShroff** [comment]
@Nadin-Nader **SCTASK0140831** is submitted for PROD deployment.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  @TracyLinder  PROD deployment is done for this ticket.

**Comment by Nadin-Nader** [comment]
@SubhaShroff Ty. I will let the Stakeholders know

**Comment by Nadin-Nader** [comment]
Sent email to Stakeholders letting them know this is on PROD. I will close on Friday 6/4/21:

> Good Morning-ish,
> 
> Hope you both are doing well. I wanted to reach out to you and let you know that this is now on AMANDA PROD and ready for use. Please verify that all is well on PROD and let me know if you see any issues. If you don’t see any issues with it, I will go ahead and close 4469 on Friday 6/4/21.
> 
> I did want to mention that the Portal side has encountered an issue with the PROD deployment. I am speaking to Martin about it and Subha is contacting the Launch IT PIPS team to see what can be done from that end. Once I have more info I will let you know.
> 
> Respectfully,
> 
> Nadin A. Nader (He/Him/His)
> IT Business Systems Analyst
> Data and Technology Services

**Comment by SubhaShroff** [comment]
@Nadin-Nader a request SCTASK0141271 is submitted to UNDO the PROD and informed Martin as well. 
Reason: Portal related request #6021 is on hold from PROD deployment.

**Comment by TracyLinder** [comment]
@Nadin-Nader will you please let Martin know that we cannot implement our FT22 fee updates until this goes to PROD.

**Comment by Nadin-Nader** [comment]
Spoke with Subha and Martin and we are able to get this and the portal side going again. We will need to reconfigure the commits for #6021 and one for this one.

> [9:17 AM] Shroff, Subhashini - Vendor
>     Thanks for reminding. When I submit request for these new commits of #6021, I will submit another one for 4469.
> Edited​[9:19 AM] Shroff, Subhashini - Vendor
>     All those commits yesterday I did with Martin's help. I need to see that fits with related Web Services's  queries. If anything is missing, I might need to do one more commit.
> ​[9:19 AM] Shroff, Subhashini - Vendor
>     And then I can submit a deployment request.

**Comment by Nadin-Nader** [comment]
Latest commit- 0e10ba55123e approved. Next step, PROD.

**Comment by SubhaShroff** [comment]
### Status:
Waiting for #6021 to reach TEST stage. Then, I will submit a new request to PROD deploy the reverted commit.

**Comment by Nadin-Nader** [comment]
Still waiting on #6021

**Comment by SubhaShroff** [comment]
### Status:
Waiting on #6021 to get to TEST.

**Comment by TracyLinder** [comment]
#6021 has been approved for PROD deployment

**Comment by SubhaShroff** [comment]
PROD Deployment Request is submitted for #4469 & #6021 - **SCTASK0144586**.

**Comment by SubhaShroff** [comment]
@TracyLinder  @Nadin-Nader  PROD Deployment is done. Needs verification.  🎉

**Comment by Nadin-Nader** [comment]
Email sent to Stakeholders:

**Comment by SubhaShroff** [comment]
### Status:
On PROD.  Waiting to hear from Stakeholders to close it.

**Comment by SubhaShroff** [comment]
### Status:
PROD deployment is done, two weeks ago.

**Comment by Nadin-Nader** [comment]
Closing today

**Comment by Nadin-Nader** [comment]
Closing. Good to go Per Kim Just have to add Subtype "Other City Department" (see #6562)
